feat: rely on Nx Parallel env var (#2561)
AmadeusITGroup/otter: CONTRIBUTING.md

> Only Otter Team members can write Remote Cache on Nx Cloud, logged-in users will be able to use Remote Cache in readonly mode.
> The Local Cache is available for all users (logged or not to Nx Cloud).

#### Increase the number of parallel process

> Per default the number of tasks run in parallel is 3 (see [Nx Documentation](https://nx.dev/recipes/running-tasks/run-tasks-in-parallel)).

Reviewer: ```suggestion
By default, the number of tasks that can run in parallel is 3 (see [Nx Documentation](https://nx.dev/recipes/running-tasks/run-tasks-in-parallel)).
```

> This number can be increase in the developer machine thanks to the [NX_PARALLEL](https://nx.dev/reference/environment-variables) environment variable by using the following command:

Reviewer: ```suggestion
This number can be increased in the developer machine thanks to the [NX_PARALLEL](https://nx.dev/reference/environment-variables) environment variable by using the following command:
```

```bash
yarn print:nx-parallel >> .env
```

> [!NOTE]

> > The command will set this number to maximum possible for your machine, it can be adapted manually after generation

Reviewer: ```suggestion
> The command will set this number to the maximum possible value for your machine, it can be adapted manually after generation
```
